### DISCUSSION TITLE
Request for statistics-graph card that responds to period selector

### DISCUSSION THREAD

**Discussion by rrozema** [flixlix/energy-period-selector-plus#19] [Ideas]
Hi, I do realise this request is not directly related to the energy-period-selector-plus card itself, but it seems that you are one of the very few who actually know how the selector mechanism works, I so far can't find any details on it anywhere. So that's why I still ask my question here...

Would it be possible for you to also create an improved variant of the [statistics-graph card](https://www.home-assistant.io/dashboards/statistics-graph/) that (additionally?) takes your energy-period-selector-plus card (and preferably the standard energy-date-selection card as well of course) for the time period to show in the graph? I think having such an improved statistics-graph card would make your awesome selector-plus card even more useful.

For example I currently use your energy-period-selector-plus card in combination with [MindFreeze's sankey-chart](https://github.com/MindFreeze/ha-sankey-chart), which makes for an awesome combination in being able to dynamically selecting which information to show. Below picture for example shows me getting all energy used plus where it went for the period over my current energy provider's contract period (started Jan 27th):
![image](https://github.com/flixlix/energy-period-selector-plus/assets/1747982/0a204d47-b940-4781-90c1-5df81d446b3b)
I would very much like to add a statistics-graph below this that shows for example the costs charged over that same period. However the existing statistics-chart only allows for configuring a fixed period :-(.

**Comment by flixlix** [comment]
Hey there, thank you very much for your request, maybe I'll create a custom card for Statistics Graph, but atm I'm very busy...
